recipe/meta.yaml

```diff
@@ -26,6 +26,7 @@ requirements:
     - python
     - pip
     - setuptools >=40.0
+    - py
```

Comment:
```suggestion
```
I would recommend to update to the latest v0.12. 
`py` is only a test dependency inside the f-strings https://github.com/search?q=repo%3Apytest-dev%2Fpytest-rerunfailures%20py.&type=code

Reply:
I've updated the version and moved `py` to test.

Reply:
Will it fail without `py` in `test/requires`?

Reply:
It did not fail without `py` in `test/requires`, but it does seem to require it in upstream.

From a grep I've done:
`./pytest-rerunfailures/test_pytest_rerunfailures.py: import py`

Figured better safe then sorry.